manifests/provider/ldap.pp

```diff
@@ -169,7 +169,8 @@
   Optional[Integer[0]]                  $ldap_idmap_range_size             = undef,
   Optional[String]                      $ldap_idmap_default_domain_sid     = undef,
   Optional[String]                      $ldap_idmap_default_domain         = undef,
-  Boolean                               $ldap_idmap_autorid_compat         = false
+  Boolean                               $ldap_idmap_autorid_compat         = false,
+  Stdlib::Absolutepath                  $conf_file_path                    = simplib::lookup('sssd::conf_file_path', { 'default_value' => '/etc/sssd/sssd.conf' })
```

Comment:
just a note, all of our defaults that are simplib::lookup() break the current PE console because they get serialized to strings in the current release (2016.5)